Makefile

```diff
@@ -97,12 +101,13 @@ $(R_FILE): $(RES_FILES) $(MANIFEST_FILE)
 
 # Special font
 
-SPECIAL_FONT_GLYPHS = $(wildcard srcs/special_font/*.svg)
-SPECIAL_FONT_SCRIPT = srcs/special_font/build.pe
+CURRENT_DIRECTORY := $(shell pwd)
+SPECIAL_FONT_GLYPHS = $(wildcard $(CURRENT_DIRECTORY)/srcs/special_font/*.svg)
+SPECIAL_FONT_SCRIPT = $(CURRENT_DIRECTORY)/srcs/special_font/build.pe
 
 _build/assets/special_font.ttf: $(SPECIAL_FONT_SCRIPT) $(SPECIAL_FONT_GLYPHS)
 	mkdir -p $(@D)
-	fontforge -lang=ff -script $(SPECIAL_FONT_SCRIPT) $@ $(SPECIAL_FONT_GLYPHS)
+	fontforge -lang=ff -script $(SPECIAL_FONT_SCRIPT) $(CURRENT_DIRECTORY)/$@ $(SPECIAL_FONT_GLYPHS)
```

Comment:
Why this change ?

Reply:
I think because fontforge is an AppImage now, files passed to it as arguments are ?mounted? somehow in a weird way, and relative paths doesn't work well.

See my test runs, the logs when the build fails
https://github.com/sdrapha/Unexpected-Keyboard/runs/5688302619?check_suite_focus=true
![image](https://user-images.githubusercontent.com/25059996/160254529-d4396902-b1a3-41b4-909b-1eabea970c0d.png)

So using absolute paths fixed the issue

Reply:
I think this is a bug in the packaging of Fontforge. I don't know if AppImage is intended to package CLI programs but paths must be relative. Are there an other prebuilt package we can use ? (perhaps from Ubuntu's store?)
I've tried using Nix and caching `/nix`. It doesn't seem to be working as I expect and I fear there can be complications. It builds in less than a minute, so still a lot more than your solution.

Reply:
I'm out of ideas, the appimage is the official distribution method that fontforge recommends. I don't know about alternative packages.
The paths are still relative, they are just expanded to absolute paths at execution. Is there a problem on that? Really asking out of curiosity, I really don't know what could go wrong, It seems ok so far though.

Reply:
It seems that I will have to accept the workaround then. You can use `$(CURDIR)` instead, which is the current directory and is automatically set by make.

I'll merge this weekend.